lte/gateway/c/oai/include/amf_config.h

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*****************************************************************************
+  Source      amf_message.h
+  Date        2020/07/28
+  Author      Ashish Prajapati
+  Subsystem   Access and Mobility Management Function
+  Description Defines Access and Mobility Management Messages
+*****************************************************************************/
+#pragma once
+
+#include <pthread.h>
+#include <stdint.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include "amf_default_values.h"
+#include "common_types.h"
+#include "3gpp_23.003.h"
+#include "3gpp_24.008.h"
+#include "log.h"
+#include "service303.h"
+
+#define MIN_GUMMEI 1
+#define MAX_GUMMEI 5
+#define MAX_APN_CORRECTION_MAP_LIST 10
```

Comment:
Is this needed for NGAP or AMF?

Reply:
> Please see comments around code style. Also double-check
> 
> * `amf_config.h` is part of this PR but no `amf_config.c`, can you mention the PR where it is included?
> * is `amf_app_messages_def.h` needed in this PR or it can be part of AMF PR.

Removed file amf_app_messages_def.h,  amf_config.h is required in this PR, amf_config.c is part of amf PR#4595